A3

```diff
@@ -0,0 +1,61 @@
+"""
+Hey, is was asked to implement the following engineering specification:
+
+Write a function that when given a tic-tac-toe board (two 
+dimensional array of strings that are either 'X', 'O', or ''), 
+return the winner of the board. If there is no winner, return None.
+
+Here is the pull request.
+"""
+
+from operator import add as list_concat
+
+def get_winner(board):
+	"""
+	Returns None if no winner. Otherwise, returns "X" or "O" as winner.
+	"""
+
+	winning_lines = get_winning_lines(board)
+	for winning_line in winning_lines:
+		winning_line_set = set(winning_line)
+		if len(winning_line_set) == 1 and "" not in winning_line_set:
+			return winning_line[0]
+	return None
+
+def get_winning_lines(board):
+	N_ROWS = len(board)
+
```

Comment:
consider using a term other than "winning lines" to describe the candidates for winning lines; it's easy to, on a first read, think these are meant to be lines which HAVE wins in them